src/components/common/dashboard/Dashboard.vue

```diff
@@ -0,0 +1,324 @@
+                    },
+                    state: OperationStatus.REJECTED,
+                    reason: "Mocked Reason",
+                    existingApprovals: {
+                        users: [],
+                        groups: [],
+                    } as ApprovalList,
+                    missingApprovals: {
+                        users: [],
+                        groups: ["Admin"],
+                    } as ApprovalList,
+                } as Operation,
+                {
+                    operationId: "FinishedMatriculation1",
+                    initiator: "MockUser4",
+                    initiatedTimestamp: "2011-10-05T14:48:00.000Z",
+                    lastModifiedTimestamp: "2020-10-05T16:48:00.000Z",
+                    transactionInfo: {
+                        contractName: UC4Identifier.CONTRACT_MATRICULATION,
+                        transactionName: UC4Identifier.TRANSACTION_ADD_MATRICULATION,
+                        parameters: "SS2020 Computer Science v3",
+                    },
+                    state: OperationStatus.FINISHED,
+                    reason: "",
+                    existingApprovals: {
+                        users: [],
+                        groups: ["Admin"],
+                    } as ApprovalList,
+                    missingApprovals: {
+                        users: [],
+                        groups: [],
+                    } as ApprovalList,
+                } as Operation,
+            ];
+
+            let mockedWatchedOps = [
+                {
+                    operationId: "FinishedMatriculation1",
+                    initiator: "MockUser4",
+                    initiatedTimestamp: "2011-10-05T14:48:00.000Z",
+                    lastModifiedTimestamp: "2020-10-05T16:48:00.000Z",
+                    transactionInfo: {
+                        contractName: UC4Identifier.CONTRACT_MATRICULATION,
+                        transactionName: UC4Identifier.TRANSACTION_ADD_MATRICULATION,
+                        parameters: "SS2020 Computer Science v3",
+                    },
+                    state: OperationStatus.FINISHED,
+                    reason: "",
+                    existingApprovals: {
+                        users: [],
+                        groups: ["Admin"],
+                    } as ApprovalList,
+                    missingApprovals: {
+                        users: [],
+                        groups: [],
+                    } as ApprovalList,
+                } as Operation,
+            ];
+
+            onBeforeMount(async () => {
+                await refresh();
+                await getEnrollmentId();
+                await getRole();
+            });
+
+            const isAdmin = computed(() => {
+                return role.value == Role.ADMIN;
+            });
+
+            async function refresh() {
+                busy.value++;
+                store.commit(MutationTypes.CLEAR_TREATED_OPERATIONS);
```

Comment:
instead of treated: processed?

Reply:
Changed with https://github.com/upb-uc4/ui-web/pull/785/commits/74593bf36f97ddaf9f89f9b4a95340cc1dd9cf8b